ss58-js/mod.test.ts

```diff
@@ -0,0 +1,132 @@
+import * as base58 from "../deps/std/encoding/base58.ts";
+import { assertEquals, assertThrows } from "../deps/std/testing/asserts.ts";
+import * as p from "../test-util/pairs.ts";
```

Comment:
Doing
```
import * as t from "../test-util/mod.ts";
```

Output the following errors
```
error: TS2322 [ERROR]: Type 'Error' is not assignable to type 'SmoldotClient<Config_> | FailedToStartSmoldotError | FailedToAddChainError'.
  Property 'inner' is missing in type 'Error' but required in type 'FailedToAddChainError'.
    return smoldotInstance;
    ~~~~~~~~~~~~~~~~~~~~~~~
    at file:///Users/parity/ghq/github.com/paritytech/capi/rpc/providers/smoldot.ts:18:5

    'inner' is declared here.
      constructor(readonly inner: unknown) {
                  ~~~~~~~~~~~~~~~~~~~~~~~
        at file:///Users/parity/ghq/github.com/paritytech/capi/rpc/providers/smoldot.ts:90:15

TS7006 [ERROR]: Parameter 'response' implicitly has an 'any' type.
      jsonRpcCallback: (response) => {
                        ~~~~~~~~
    at file:///Users/parity/ghq/github.com/paritytech/capi/rpc/providers/smoldot.ts:25:25

TS2503 [ERROR]: Cannot find namespace 'smoldot'.
  #chain?: smoldot.Chain;
           ~~~~~~~
    at file:///Users/parity/ghq/github.com/paritytech/capi/rpc/providers/smoldot.ts:38:12

TS2503 [ERROR]: Cannot find namespace 'smoldot'.
const _state: { smoldotInstance?: smoldot.Client } = {};
                                  ~~~~~~~
    at file:///Users/parity/ghq/github.com/paritytech/capi/rpc/providers/smoldot.ts:74:35

TS2503 [ERROR]: Cannot find namespace 'smoldot'.
async function ensureInstance(): Promise<smoldot.Client | FailedToStartSmoldotError> {
                                         ~~~~~~~
    at file:///Users/parity/ghq/github.com/paritytech/capi/rpc/providers/smoldot.ts:76:42

Found 5 errors.
```

I'll investigate

cc @harrysolovay

Reply:
I can't reproduce those errors locally or [on CI](https://github.com/paritytech/capi/commit/ad78b161b9ab9e07440a1e7cafa9ba606bbedc07), so I think this is a local issue.

Can you try running with `--reload` and/or nuking `DENO_DIR` (from `deno info`)?

Reply:
My issue was that I was running the tests with
```
deno test ss58/
```
instead of 
```
deno task test ss58/
```
which runs
```
deno test -A --no-check=remote -L=info --ignore=target "ss58/"
```